src/navinator.js

```diff
@@ -25,26 +26,35 @@
             var _ = this;
             _.defaults = {
                 buttonElement: '#navinator-button',
+                debug: false,
                 direction: 'left',
                 minWidth: 0,
-                navZ: 'bottom',
-                pageElement: '#page-container'
+                pageElement: '#page-container',
+                subNavElement: false
             };
 
             _.options = $.extend({}, _.defaults, settings);
 
             // Navinator Globals
+            _.el = element;
             _.$navElement = $(element);
             _.$pageElement = $(_.options.pageElement);
             _.$buttonElement = $(_.options.buttonElement);
             _.navCreated = false;
+            if (_.options.subNavElement) {
+                var subNavClassOrID = _.options.subNavElement.charAt(0);
+                var subNavEl = _.options.subNavElement.substring(1);
+                _.subNav = subNavClassOrID + 'navinator-' + subNavEl;
+            } else {
+                _.subNav = _.options.subNavElement;
+            }
 
             // Check if the page wrapper and button elements are present
-            if (_.$pageElement.length === 0) {
+            if (_.$pageElement.length === 0 && _.options.debug) {
                 console.error('%cNavinator Error: pageElement doesn\'t exist', 'font-weight: bold');
-            } else if (_.$buttonElement.length === 0) {
+            } else if (_.$buttonElement.length === 0 && _.options.debug) {
                 console.error('%cNavinator Error: buttonElement doesn\'t exist', 'font-weight: bold');
-            } else {
+            } else if (_.$pageElement.length !== 0 && _.$buttonElement.length !== 0) {
                 _.init();
             }
```

Comment:
Checking if debug is turned on to display console messages. Then checks to see if there are the necessary elements are present to initialize the script.